cmd/oci-runtime-tool/generate.go

```diff
@@ -103,7 +103,7 @@ var generateFlags = []cli.Flag{
 	cli.StringSliceFlag{Name: "process-rlimits-add", Usage: "specifies resource limits for processes inside the container. "},
 	cli.StringSliceFlag{Name: "process-rlimits-remove", Usage: "remove specified resource limits for processes inside the container. "},
 	cli.BoolFlag{Name: "process-rlimits-remove-all", Usage: "remove all resource limits for processes inside the container. "},
-	cli.BoolFlag{Name: "process-tty", Usage: "allocate a new tty for the container process"},
+	cli.BoolFlag{Name: "process-terminal", Usage: "specifies terminal attach to the process"},
```

Comment:
It sounds like setting a terminal with detailed info (size for example).
We can just reuse the word in the spec file: `specifies whether a terminal is attached to the process`.

Reply:
updated.